frontend/src/workspace/buildingTools/ValNode.js

```diff
@@ -335,7 +399,7 @@ function ValNode(props) {
       <Circle
         x={valueWidth}
         y={valueWidth/2}
-        radius={valueWidth/8}
+        radius={8}
```

Comment:
number?

Reply:
I'm not certain where this number came from, but it should definitely be a ratio instead.